nav2_controller/include/nav2_controller/nav2_controller.hpp

```diff
@@ -42,6 +43,8 @@ class ProgressChecker;
 class ControllerServer : public nav2_util::LifecycleNode
 {
 public:
+  typedef std::unordered_map<std::string, nav2_core::LocalPlanner::Ptr> ControllerMap;
```

Comment:
Alternatively (modern style?) `using ControllerMap = std::unordered_map<std::string, nav2_core::LocalPlanner::Ptr>;`

Reply:
Using would be fine, can do